x-pack/plugins/uptime/server/lib/requests/get_journey_screenshot.test.ts

```diff
@@ -26,7 +26,7 @@ describe('getJourneyScreenshot', () => {
     expect(
       await getJourneyScreenshot({
         uptimeEsClient: mockSearchResult([], {
-          // @ts-expect-error incomplete search result
+          // @ts-expect-error incomplete object in test
```

Comment:
There are two situations in which I do this. If I need to use a "core" type of interface that has a lot of irrelevant fields on it, or if it's a type of an object where only one or two fields are referenced in the function being tested. I think it can help keep the test focused for people who are trying to understand what it's doing, if there are complex objects in a test it can be tougher to quickly glean what's happening.

That being said, I did use these comments a lot in this PR, so I can go back and do a little more mocking to see if I can remove most/all of them if you think it'll add value to the tests.

Reply:
Fixed: https://github.com/elastic/kibana/pull/103979/commits/4259dd5847b50ee57677fa134cd268f632c1aad8